doc/ug/getting_started_verilator.md

```diff
@@ -15,7 +15,7 @@ _Make sure you followed the install instructions to [prepare the system]({{< rel
 
 ## Simulating a design with Verilator
 
-First the simulation needs to built itself.
+First the simulation needs to build itself.
```

Comment:
It's hard to say what was originally intended, but the simulation doesn't build itself -- the simulation (itself) is being built. Maybe you can update the wording to reflect that?